neqo-http3/src/lib.rs

```diff
@@ -321,6 +323,15 @@ pub trait RecvStream: Debug {
     fn done(&self) -> bool;
     fn stream_type(&self) -> Http3StreamType;
     fn http_stream(&mut self) -> Option<&mut dyn HttpRecvStream>;
+    fn priority(&self) -> Priority;
+
+    /// only relevant for streams supporting priority updates
+    #[allow(unused_variables)]
```

Comment:
You shouldn't need default implementations for these if you do as Dragana suggests.  Which means that you don't need to suppress warnings (you should avoid doing that anyway; renaming `priority` to `_priority` would work here).